test/browser/tests/step_implementation.js

```diff
@@ -0,0 +1,26 @@
+/* globals beforeSuite, afterSuite */
+const { $, waitFor, openBrowser, currentURL, closeBrowser, text } = require('taiko')
+const assert = require('assert')
```

Comment:
would it be possible to use chai instead of assert?

Reply:
done